src/jayashri/Test1_Prg1.java

```diff
@@ -0,0 +1,22 @@
+package jayashri;
+
+class Test1_Prg1{
+
+	void nonRepeating(String name){
+		for (int i = 0; i < name.length(); i++){
+            int count = 0;
+		for (int j = 0; j < name.length(); j++){
+            if (name.charAt(i) == name.charAt(j))
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop